ui/web/WEB-INF/templates/material/components/composition.xhtml

```diff
@@ -25,60 +25,99 @@
+                                    style="border-width: 0px"
+                                    value="#{msgs.materialCreation_composition_chooseTypeLabel}">                                    
+                                </p:outputLabel>
+                            </p:column>
+                            <p:column
+                                styleClass="borderless compositionTypeSelectBox"
+                                style="width:60%"
+                                >
+                                <b:selectOneMenu
+                                    ajax="true"
+                                    disabled="#{!materialCompositionBean.isCompositionTypeEditable()}"
+                                    update="@(.resultTable) @(.searchTabView)"
+                                    value="#{materialCompositionBean.choosenType}">
+                                    <f:selectItems 
+                                        value="#{materialCompositionBean.getCompositionTypes()}"
+                                        itemLabel="#{materialCompositionBean.getLocalizedCompositionType(type)}"
+                                        var="type"/>
+                                </b:selectOneMenu>                            
+                            </p:column>
+                        </p:row>
                     </p:panelGrid>
 
-                    <p:panelGrid>
-                        <p:row>
-                            <p:column>
+                    <p:panelGrid
+                        class="wideBody"
+                        id="searchPanelId">
+                        <p:row styleClass="borderless">
+                            <p:column 
+                                style="vertical-align: top;border-top-width: 0px;border-bottom-width: 0px;border-left-width: 0px;border-left-right: 2px;">
                                 <p:panelGrid>
-                                    <p:row>
-                                        <p:column>
+                                    <p:row styleClass="borderless">
+                                        <p:column styleClass="borderless">
                                             <p:inputText
+                                                style="width: 450px"
+                                                styleClass="inputText"
                                                 value=""
-                                                placeholder="Suchbegriff">
-                                            </p:inputText>            
+                                                placeholder="#{msgs.materialCreation_composition_nameSearchPlaceholder}">
+                                                </p:inputText>            
                                         </p:column>
-                                        <p:column>
+                                        <p:column styleClass="borderless">
                                             <p:commandButton
```

Comment:
this commandbutton needs to process the previous inputtext and the inputs in the tabview (maybe by id "searchTabViewId")